src/bundler/bundle_v2.zig

```diff
@@ -3748,7 +3759,7 @@ pub const JSMeta = struct {
 };
 
 pub const Graph = struct {
-    // TODO: consider removing references to this in favor of bundler.options.code_splitting
```

Comment:
removing this comment is incorrect. i had put off deciding if removing code_splitting from this structure was possible, but i just checked again and i realize it can be removed. please don't remove TODO tasks without actually performing them.